index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    <title>Somarib Portfolio</title>
+
+    <link rel="stylesheet" href="stylesheet.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="maincont">
+
+      <header class="app-bar">
+        <nav class="lower-bar">
+          <p class="logoo">Somarib Emmanuel</p> 
```

Comment:
- [ ] As the previous code reviewer mentioned, the logo should be involved in an `<a>` tag. Kindly change that, this is for a better UX, convention, and semantics reasons.